app/javascript/src/components/Dashboard/Notes/Card.jsx

```diff
@@ -20,15 +20,20 @@ const Card = ({ title, description, assignedContactId, tag, created_at }) => {
             {description}
           </p>
           <div className="flex w-full items-center justify-between">
-            <Tag className="mt-2" label={tag} />
+            <Tag className="mt-2" label={tag.value} />
             <p className="mt-2 flex items-center text-sm text-gray-600">
               <Clock className="mr-1 h-4 w-4" />
-              {moment.utc(created_at).local().startOf("seconds").fromNow()}
+              <Tooltip
+                content={moment(created_at).format("dddd, h:mm a")}
```

Comment:
Move the formatting to a separate utils function with a proper name.
Also, use dayjs over other libraries and inbuilt functions: https://courses.bigbinaryacademy.com/learn-react/react-best-practices/miscellaneous-tips/#prefer-dayjs-over-inbuilt-date-functions